sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2162,6 +2296,65 @@ public partial class EventDataInfo : System.ClientModel.Primitives.IJsonModel<Az
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Monitor.Models.ExporterType left, Azure.ResourceManager.Monitor.Models.ExporterType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Monitor.Models.ExporterType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Monitor.Models.ExporterType left, Azure.ResourceManager.Monitor.Models.ExporterType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ExtendedLocation : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.ExtendedLocation>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.ExtendedLocation>
+    {
+        public ExtendedLocation(Azure.ResourceManager.Monitor.Models.ExtendedLocationType extendedLocationType, string name) { }
+        public Azure.ResourceManager.Monitor.Models.ExtendedLocationType ExtendedLocationType { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        Azure.ResourceManager.Monitor.Models.ExtendedLocation System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.ExtendedLocation>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.ExtendedLocation>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Monitor.Models.ExtendedLocation System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.ExtendedLocation>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.ExtendedLocation>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.ExtendedLocation>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ExtendedLocationType : System.IEquatable<Azure.ResourceManager.Monitor.Models.ExtendedLocationType>
```

Comment:
same issue here, but if the above issue is resolved and we no longer generate the `ExtendedLocation` type, this type will be gone as well.

Reply:
Resolved in latest update.